.eslintrc.yml

```diff
@@ -86,7 +86,9 @@ rules:
     - error
     - functions: false
   no-undef: 2
-  no-unused-vars: 2
+  no-unused-vars:
+    - 2
+    - varsIgnorePattern: "^_"
```

Comment:
What does this do?

Reply:
ignores unused variables whose name stars with underscore. Which ATM is used like this:
https://github.com/osbuild/image-builder-frontend/pull/56/files#diff-edcaaaf245bb6bcb9867fb52970fce66cdc1b83ca466911811c9ddb914212842R80

b/c I couldn't figure out how to assign only to the `history` variable ignoring everything else from the return value.